### PR TITLE
fix: postgres set max parallel workers per gather to 8

### PR DESCRIPTION
**Pull request by vasco-santos**

### CODE REVIEW

File: packages/api/scripts/cmds/db-sql.js [hunk 1]
```diff
@@ -25,6 +25,12 @@ export async function dbSqlCmd(opts) {
   })
 
   // read all the SQL files
+  const configSql = fs.readFileSync(
+    path.join(__dirname, '../../postgres/config.sql'),
+    {
+      encoding: 'utf-8',
+    }
```
Comment (alanshaw) [inline packages/api/scripts/cmds/db-sql.js:32]: 🙏 please can we make all these a little less verbose and use `'uft8'` instead of `{ encoding: 'utf-8' }`?

**Review by alanshaw — APPROVED** [review]
LGTM, one small request though...

File: packages/api/db/config.sql [hunk 2]
```diff
@@ -0,0 +1 @@
+SET max_parallel_workers_per_gather TO 8;
```
[the remark this answers is not in the document]
Comment (alanshaw) [reply packages/api/db/config.sql:1]: That's `max_parallel_workers`, `max_parallel_workers_per_gather` is set to 2 currently in both production databases.

**Review by alanshaw — APPROVED** [review]

File: packages/api/db/config.sql (hunk 2 above)
[the remark this answers is not in the document]
Comment (vasco-santos) [reply packages/api/db/config.sql:1]: lol my bad, gonna revert back to 4 then until we feel a need for 8

**Pull request merged**